app/index.html

```diff
@@ -36,5 +36,7 @@
     <script src="assets/travis.js"></script>
 
     {{content-for "body-footer"}}
+
+    <a href="mailto:support@travis-ci.com" title="Ask Travis CI support for help" id="helpscout-link" class="helpscout-link">Do you have a question?</a>
```

Comment:
I’d usually expect this to be a component, which would let you wrap up the click-handling and show/hiding in a more Ember-y fashion. Buuuut maybe I am too focused on Ember conventions??? 😬